### PR TITLE
Update embedding docs with correct steps for embedding into iframe

### PR DESCRIPTION
**Pull request by svelle**
Fixes https://github.com/mattermost/docs/issues/3106

Might need to make a commit or two more to fix the code highlighting

### CODE REVIEW

**Comment by DSchalla** [comment]
Pinged @svelle on Community, we will do some more testing and get back to this afterwards.

**Comment by justinegeffen** [comment]
Hi @DSchalla! Wanted to check in on this and find whether you've had a chance to do the testing and if there's anything I can assist with. :)

**Comment by DSchalla** [comment]
@svelle IIRC you wanted to look into this after our last discussion, do you remember the status of this?

**Comment by svelle** [comment]
@DSchalla oops, this kinda went under. sorry about that. Let me read that up again and see where I was.

**Comment by jasonblais** [comment]
@svelle I saw someone asking about embedding Mattermost, this will be a great reference once merged :)

**Comment by justinegeffen** [comment]
Following up on this in light of @jasonblais's comment. Let me know if I can help push this over the line. :)

File: source/integrations/embedding.rst [hunk 1]
```diff
@@ -49,22 +49,25 @@ Save the below HTML code in a file called ``mattermost-button-example.html`` the
...
   </html>
 
-
 Embedding Mattermost in web applications using an iframe  
 ----------------------------------------------------------
 
 .. warning:: Any web application embedded into another using an iframe is at risk of security exploits, since the outer application intercepts all user input into the embedded application, an exploit known as `Click-Jacking <https://en.wikipedia.org/wiki/Clickjacking>`__. By default, Mattermost disables embedding. If you choose to embed Mattermost using the following instructions we highly recommend it is done only on a private network that you control. 
 
-To embed Mattermost in an iframe by overriding Mattermost security features: 
+To embed Mattermost in an iframe update your `NGINX configuration<https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server>` to strip out the security policy settings in the HTTP header.
```
Comment (justinegeffen) [inline source/integrations/embedding.rst:57]:
```suggestion
To embed Mattermost in an iframe update your `NGINX configuration <https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server>`_ to strip out the security policy settings in the HTTP header.
```

File: source/integrations/embedding.rst [hunk 2]
```diff
@@ -49,22 +49,25 @@ Save the below HTML code in a file called ``mattermost-button-example.html`` the
...
+
+    Replace all occurences of the following line in your proxy config:
```
Comment (justinegeffen) [inline source/integrations/embedding.rst:59]:
```suggestion
    Replace all occurrences of the following line in your proxy config:
```

**Comment by justinegeffen** [comment]
Following up on this, @svelle. I made some minor editorial changes. Wondering if this is good to go, based on the detailed disclaimer preceding the changes. LMK. :)

**Comment by svelle** [comment]
@justinegeffen how about we move this into a forum post. It's not officially recommended to embed mattermost anyways and we don't really "support" it.

**Comment by justinegeffen** [comment]
I'm happy with that, thanks @svelle.

**Pull request closed without merging**